.github/workflows/release.yml

```diff
@@ -76,7 +95,7 @@ jobs:
       # this needs `--no-verify-access` until https://github.com/lerna/lerna/issues/2788 is fixed
       - name: Release to npm
         run: |
-          $(npm bin)/lerna publish from-package --yes --dist-tag ${GITHUB_REF##*/} --pre-dist-tag ${GITHUB_REF##*/} --no-verify-access
+          $(npm bin)/lerna publish from-package --yes --dist-tag ${TAG} --pre-dist-tag ${TAG} --no-verify-access
```

Comment:
@cds-amal how does this line look to you?

Reply:
IT looks ok to me as long as lerna is installed :) how did the action run?

Reply:
[It ran](https://github.com/trufflesuite/ganache/runs/4859668937?check_suite_focus=true)! This step wasn't included in the test action because it has side effects.